double_entry.gemspec

```diff
@@ -17,6 +17,31 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ['lib']
 
+  gem.post_install_message = <<-'POSTINSTALLMESSAGE'
+Please note the following changes in DoubleEntry version 10.4:
```

Comment:
we are *now* this will be a problem in the next release. so whats our next release? minor or major?